src/essence/Tools/Measure/MeasureTool.js

```diff
@@ -43,10 +44,10 @@ const Measure = () => {
             .on('click', MeasureTool.clickMap)
             .on('mousemove', MeasureTool.moveMap)
             .on('mouseout', MeasureTool.mouseOutMap)
-        Globe_.litho
-            .getContainer()
-            .addEventListener('click', MeasureTool.clickGlobe, false)
-            .addEventListener('mousemove', MeasureTool.moveGlobe, false)
+        // Globe_.litho
+        //     .getContainer()
+        //     .addEventListener('click', MeasureTool.clickGlobe, false)
+        //     .addEventListener('mousemove', MeasureTool.moveGlobe, false)
 

```

Comment:
There is a merge conflict that needs to be resolved here.
```
        const globeCont = Globe_.litho.getContainer()
        globeCont.addEventListener('click', MeasureTool.clickGlobe, false)
        globeCont.addEventListener('mousemove', MeasureTool.moveGlobe, false)
```

Reply:
Yep, the (now) existing `const globeCont = Globe_.litho...` should be good.